lr-cli/cortx_setup/commands/pillar_sync.py

```diff
@@ -51,8 +51,11 @@ def run(self, **kwargs):
         conf_path = str(PRVSNR_FACTORY_PROFILE_DIR / 'confstore')
         # backup local consftore data
         self.logger.debug(f"Copy local confstore file to {conf_path}")
-        state = 'components.provisioner.confstore'
-        StatesApplier.apply([state])
+        state_create = 'components.provisioner.confstore_dir_create'
+        StatesApplier.apply([state_create], targets=local_minion_id())
+
+        state_copy = 'components.provisioner.confstore_copy'
```

Comment:
Lets change this to:
`conf_copy = 'components.provisioner.confstore_copy'`

Reply:
changed var name